src/core/server/capabilities/resolve_capabilities.test.ts

```diff
@@ -36,7 +36,7 @@ describe('resolveCapabilities', () => {
   });
 
   it('returns the initial capabilities if no switcher are used', async () => {
```

Comment:
Nit: I noticed none of these tests have assertions for what args are passed to the switchers. I know this isn't your code though so I don't expect you to change it.